docassemble/AssemblyLine/data/questions/ql_baseline.yml

```diff
@@ -1109,4 +1109,19 @@ help:
 code: |
   # Every form has a user, at least as our default. You can override this
   # by adding your own question or code later in the file
-  users.there_are_any = True
\ No newline at end of file
+  users.there_are_any = True
+---
+code: |
+  # This default overflow message is only used in the constructor for ALDocument
+  al_default_overflow_message = " [See addendum]"
+---
+# You can override this addendum with a custom one
+generic object: ALDocument
+attachment:
+  variable name: x.addendum
+  docx template file: al_basic_addendum.docx
+---
+generic object: ALAddendumField
+code: |
+  x.label = x.field_name.capitalize().replace("_", " ")
```

Comment:
Totally right about removing Commonwealth of Massachusetts. Ultimately it may make sense to make the case caption itself swappable. For now, removing it is good enough which I did in latest commit.

Yes, overflow_value() does resume the area mid-sentence. It depends quite a bit on the length of the original text whether it makes sense to reproduce it in full. But I think making it configurable is a good idea. I'll open an issue for both of these suggestions.